packages/components/src/components/Combobox/Combobox.stories.tsx

```diff
@@ -0,0 +1,243 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import React, { useEffect, useState } from "react";
+import toLower from "lodash/toLower";
+import includes from "lodash/includes";
+import map from "lodash/map";
+import { useComboboxStore } from "@ariakit/react";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+import find from "lodash/find";
+import { Box } from "../../primitives/Box";
+import { Label } from "../Label";
+import { HelpText } from "../HelpText";
+import { Button } from "../Button";
+import { Combobox, ComboboxItemProps, ComboboxProps } from "./Combobox";
+import { ComboboxInput } from "./ComboboxInput/ComboboxInput";
+import { ComboboxPopover } from "./ComboboxPopover/ComboboxPopover";
+import { ComboboxItem } from "./ComboboxItem/ComboboxItem";
+
+export default {
+  component: Combobox,
+  title: "Components/Combobox",
+} as Meta<typeof Combobox>;
+
+const items: ComboboxItemProps[] = [
+  { value: "1", label: "Option One" },
+  { value: "2", label: "Option Two" },
+  { value: "Option Three", label: "Option Three" },
+  { value: "Option Four", label: "Option Four" },
+];
+
+const Template: StoryFn<ComboboxProps> = ({ children, ...props }) => (
+  <Combobox {...props}>{children}</Combobox>
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  items,
+  placeholder: "Select an option",
+};
+Default.parameters = {
+  docs: {
+    description: {
+      story: "The combobox will filter the items based on the input value.",
+    },
+  },
+};
+
+export const WithoutFilterItem = Template.bind({});
```

Comment:
when would we not filter?

Reply:
I couldn't remember a scenario where we would not filter. I researched some design systems and a few have the "default" without the filter, that's why I added it here. But IMO, it's better to use a "Combobox" that lets you search and match items. Either way, I kept the `filterItem` prop in case we need it but removed the `WithoutFilterItem` story as it doesn’t seem necessary.